tippecanoe

```diff
@@ -16,7 +16,7 @@ tippecanoe -o rnet-dev.pmtiles \
   --maximum-tile-bytes=5000000 \
   --simplification=10 \
   --buffer=5 \
-  --force  rnet_school_commute.geojson
+  --force  combined_network_tile.geojson
```

Comment:
Do we need to change the rnet tile code in `_targets.R`?

Reply:
No